lib/components/Dice/DiceFace.dart

```diff
@@ -8,13 +8,18 @@ class DiceFace extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      color: Colors.red,
+      color: Colors.white,
       height: this.size.height,
       width: this.size.width,
       child: Center(
-        child: Text(
-          this.display.toString(),
-          style: TextStyle(color: Colors.white),
+        child: ClipRRect(
+          borderRadius: BorderRadius.circular(8.0),
+          child: Image.asset(
+            'assets/images/dice-' + this.display.toString() + '.png',
```

Comment:
can you make. it the the form of string literal 
it's easier to read that ways :) 
rest looks good to me